roles/openshift_master_facts/tasks/main.yml

```diff
@@ -76,6 +76,7 @@
       controllers_env_vars: "{{ openshift_master_controllers_env_vars | default(None) }}"
       audit_config: "{{ openshift_master_audit_config | default(None) }}"
       scheduler_args: "{{ openshift_master_scheduler_args | default(None) }}"
+      debug_level: "{{ openshift_master_debug_level | default(None) }}"
```

Comment:
I thought there was another variable for this that you can just use.

Reply:
You shouldn't have to add this here - you should be able to use it directly in the template because the role defaults it.